service/src/main/java/tds/config/web/resources/TaCheckinTimeResource.java

```diff
@@ -0,0 +1,16 @@
+package tds.config.web.resources;
+
+/**
+ * A HATEOAS representation of the TA Check-In Time value.
+ */
+public class TaCheckinTimeResource {
```

Comment:
Since I've changed how the TA check-in time is handled, this class has been removed.